### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by JoyJeeo**
I want to join your master.

### CODE REVIEW

**Comment by JoyJeeo** [comment]
123

**Comment by JoyJeeo** [comment]
hello

File: newfeature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+feature 01
+feature 02
+feature 03
+
+feature 04
```
Comment (lessJoyJeeo) [inline newfeature:5]: 5?

File: newfeature [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+feature 05
+fix 02
+
```
Comment (lessJoyJeeo) [inline newfeature:8]: 8?

**Review by lessJoyJeeo — COMMENTED** [review]
01

**Pull request merged**